src/emcpy/stats/obspace.py

```diff
@@ -0,0 +1,230 @@
+import numpy as _np
```

Comment:
@CoryMartin-NOAA 
This looks like a very specific use case.  Should this be part of `emcpy`?

Reply:
I guess this becomes a philosophical question of where the cutoff happens and what constitutes 'specific' vs 'generic'.

I see two approaches:
1) one 'synoptic' repository that contains generic tools and specific, shared applications of said tools
2) a generic tool repository and numerous, app/group/branch specific repositories that use the generic tools

I think both have their pros/cons. 1 runs the risk of being hard to manage, 2 runs the risk of duplicated efforts

Reply:
@delippi after @aerorahul 's comment and thinking about this over the past couple weeks and now with fresh eyes, I think this functionality probably belongs in PyGSI?

Reply:
@CoryMartin-NOAA I've removed this from emcpy and am working on getting it into pygsi. Thanks for the suggestion.